### PR TITLE
Replacement for PR #1068

### PR DESCRIPTION
**Pull request by madflow**
This PR replaces #1068. 
- [x] Test added
- [x] Rebased

Let me know if this will suffice.

### CODE REVIEW

**Comment by daviesgeek** [comment]
My apologies; one more quick thing. Can you please update the docs to include this change?

**Comment by madflow** [comment]
Well, I reckon the "docs" are the README, right? I added a short hint and a hopefully minimal and correct example under "Decoupled Services".

**Comment by madflow** [comment]
@daviesgeek We are using this in production for a year now. I believe it is save to merge. So how about it?

File: README.md [hunk 1]
```diff
@@ -666,7 +666,8 @@ app.controller('MainCtrl', function(Restangular, BingRestangular) {
 
 ### Decoupled Restangular Service
 
-There're some times where you want to use Restangular but you don't want to expose Restangular object anywhere. For those cases, you can actually use the `service` feature of Restangular.
+There're some times where you want to use Restangular but you don't want to expose Restangular object anywhere.
```
Comment (daviesgeek) [inline README.md]: *There are

I know this is in the original docs, but would you mind fixing this sorta typo while you're here?

**Comment by daviesgeek** [comment]
My apologies. I have not had the time to keep an eye on this project like I've wanted to :( 

Yes, you updated the correct docs. I will merge soon.

**Pull request merged**